astroquery/mast/collections.py

```diff
@@ -42,7 +42,9 @@ def __init__(self):
 
         services = {"panstarrs": {"path": "panstarrs/{data_release}/{table}.json",
                                   "args": {"data_release": "dr2", "table": "mean"}}}
-        self._service_api_connection.set_service_params(services, "catalogs", True)
+
+        self._service_api_connection.set_service_params(services, "catalogs",
+        True)
```

Comment:
revert this formatting change. I think you could avoid adding them in the future by adding a less conservative line length into your editor (e.g. 100 or 120 could work, you can always break long lines manually where they make logical sense, and then avoid reformats like this where the new one is neither more legible nor strictly following pep8 conventions)